tests/telemetry/telemetry_utils.py

```diff
@@ -139,7 +130,8 @@ def trigger_logger(duthost, log, process, container="", priority="local0.notice"
 
 def generate_client_cli(duthost, gnxi_path, method=METHOD_GET, xpath="COUNTERS/Ethernet0", target="COUNTERS_DB",
                         subscribe_mode=SUBSCRIBE_MODE_STREAM, submode=SUBMODE_SAMPLE,
-                        intervalms=0, update_count=3, create_connections=1, filter_event_regex=""):
+                        intervalms=0, update_count=3, create_connections=1, filter_event_regex="",
+                        timeout=-1):
```

Comment:
>timeout [](http://example.com/codeflow?start=24&length=7)

Could you add some function comments to explain each parameter? Especial for this one, what is the contract? What is the timeout units?

Reply:
Added comments